src/k8s_etcd.rs

```diff
@@ -176,6 +176,34 @@ impl InMemoryK8sEtcd {
         self.deleted_keys.lock().await.insert(key.to_string());
         Ok(())
     }
+
+    pub(crate) async fn update_member(&self, value: String) -> Result<()> {
+        let etcd_client = match &self.etcd_client {
+            Some(etcd_client) => etcd_client,
+            None => bail!("etcd client not configured"),
+        };
```

Comment:
`context` on an `Option` turns it into a `Result`, so we can write the error directly on it then `?` to get equivalent code:

```suggestion
        let etcd_client = self.etcd_client.context("etcd client not configured")?;
```